build/media_source/plg_media-action_focus/css/focus.css

```
@@ -0,0 +1,5 @@
+/* This is the CSS File for Smart Crop */
+#attrib-smartcrop
+{
+    overflow-y: auto
```

```suggestion
#attrib-smartcrop {
    overflow-y: auto;
```
Not sure, but I think this is the code style ...